src/store/Chat/index.js

```diff
@@ -1,15 +1,11 @@
 import API from '../endpoints';
 import makeDummy from '../dummy';
 
-// Flag to enable putting dummy data into redux
-// const DUMMY_DATA = process.env.NODE_ENV !== 'production';
-// const dummy = makeDummy(6, 50);
-
 const initialState = {
   activeId: '',
   activeIsOpen: null,
-  chats: [], // DUMMY_DATA ? dummy.chatSessions : [],
-  messages: [], // DUMMY_DATA ? dummy.messages : [],
+  chats: [],
```

Comment:
Have you pulled from master? you should pull from master ...

Reply:
yeah i know i know. Haven't had a chance to work on this for a few days haha. I'm hoping it will fix a bug in this branch where I can't send messages